src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseEventFieldComplex.java

```diff
@@ -11,15 +11,25 @@
 
     private Integer order;
 
+    private String displayContextParameter;
+
     public CaseEventFieldComplex() {
     }
 
     public CaseEventFieldComplex(String reference,
-                                 Integer order) {
+                                  Integer order) {
         this.reference = reference;
         this.order = order;
     }
 
+    public CaseEventFieldComplex(String reference,
```

Comment:
minor: 
consider introducing a builder for CaseEventFieldComplex rather than telescoping constructors:
https://www.vojtechruzicka.com/avoid-telescoping-constructor-pattern/